### ISSUE TITLE
Identity images in dataset. 重复的图片。虽然没啥影响但是发现了，分享给各位

### ISSUE THREAD

**Issue by saltfish-len** [buptlihang/CDLA#10]
There were 35 repetitions of 17 pairs of images in the training set and 28 repetitions of 14 pairs of images in the test set.
在训练集中有35张共17组图片重复，在测试集中有28张共14组图片重复。

**Train**
SHA-256哈希值为 0458d737cb0deb0889de5e4cac51bf7b8820b0fdc74aaaefdba2d9c1ac1d1992 的文件:
./train/train_2436.jpg
./train/train_3423.jpg

SHA-256哈希值为 1fc4b92708fad0e02f4ff3f2d9ff0d4f309efb6582d7e0c36e73c0cac4fe7c45 的文件:
./train/train_3190.jpg
./train/train_0689.jpg

SHA-256哈希值为 6394d1f30ed81499ef44990bdb23297a1181d939100964c58e73f55ec4fbb8e7 的文件:
./train/train_2583.jpg
./train/train_1541.jpg

SHA-256哈希值为 6cd63368e583ab920132f2db2213cd2ecba5b82f756cc536d31cb17b4f282ff5 的文件:
./train/train_2732.jpg
./train/train_2228.jpg

SHA-256哈希值为 6d89352f83b66ea17b2c8ae8e2acecbd259c1dc0af3ab25f8cc47968e22a23d4 的文件:
./train/train_2028.jpg
./train/train_4705.jpg

SHA-256哈希值为 857f302a83018207d7fd96c5caec796ec8b80f6ac573f63d11175f3f4ba870a2 的文件:
./train/train_4334.jpg
./train/train_2460.jpg

SHA-256哈希值为 99309e7dad81cd4139a63ec9b8414ce13bf327637416f6cd74be2b8f2830a2c9 的文件:
./train/train_1471.jpg
./train/train_0108.jpg

SHA-256哈希值为 b4a70bad597aeb5215b6db3d00b8c6acf187edd2c1e129e1dbf2f6128f1a0e88 的文件:
./train/train_4096.jpg
./train/train_1727.jpg

SHA-256哈希值为 bc3ce33ef15652a1a292398f8f52edfae489180cdd2125735219545443e89c02 的文件:
./train/train_4907.jpg
./train/train_2340.jpg

SHA-256哈希值为 bd3027c17646b61982800328a15e8c2440e42b66fd815c227a3f830a41e69a5c 的文件:
./train/train_0081.jpg
./train/train_3362.jpg

SHA-256哈希值为 c36dd61639568d56c5abca7a5c4ce05754c5c1a7b9134d7c301d4c39edeb6b56 的文件:
./train/train_1328.jpg
./train/train_0195.jpg

SHA-256哈希值为 c6381f172a5004bad77a0880f513134b8f0ba0569313f55eff951993a1d7cd6e 的文件:
./train/train_1176.jpg
./train/train_1318.jpg

SHA-256哈希值为 d95d982292f78c62ec84ec11f279a42ebcbce44d59e34356846d3eb7d33b7173 的文件:
./train/train_3389.jpg
./train/train_4471.jpg

SHA-256哈希值为 ec806cd3df73d84a11f290830a12d2f441a48913198b738246dcc6434e6d27cd 的文件:
./train/train_4290.jpg
./train/train_1684.jpg
./train/train_3247.jpg

SHA-256哈希值为 efbf4b134d20b76cd6b2c3655f2e44cf68eaa382a7c4be2aea5b237f33308405 的文件:
./train/train_4646.jpg
./train/train_2203.jpg

SHA-256哈希值为 f1ad77cc91fe17803b4d57741d474254e94afaf4d390a0b88a4e436d01bccfc8 的文件:
./train/train_3791.jpg
./train/train_1654.jpg

SHA-256哈希值为 f54e46f1445eff30c7e1740e7c4ed63ff5580c4e473c5a60760e84a1851a9f87 的文件:
./train/train_1835.jpg
./train/train_4942.jpg

**Validation**
SHA-256哈希值为 054c245c243af078fdf8403c878142bc892888749d0a86d175b7ea0683f16e1e 的文件:
./val/val_0146.jpg
./val/val_0227.jpg

SHA-256哈希值为 0a21b40e33b04f12a718947f94fc055506286843a86df8f94821c3d17f2874d3 的文件:
./val/val_0018.jpg
./val/val_0636.jpg

SHA-256哈希值为 0c8b97d6df0991bd1001c9d9ae5e9d1489844946cccc9c7da043fda42578683d 的文件:
./val/val_0524.jpg
./val/val_0464.jpg

SHA-256哈希值为 0f837906d0c1869068ea88ec2b6a52672aa2109933e452dfa5df04921c90d24e 的文件:
./val/val_0300.jpg
./val/val_0991.jpg

SHA-256哈希值为 18fdc0b882df7193c26e0d8db97f3a20a1bf98405c1e769d28b6fde0b38acd70 的文件:
./val/val_0657.jpg
./val/val_0984.jpg

SHA-256哈希值为 1b09feeae105d69dcce53246b83261513a1c33f21dadce028c5fdc1aeae8391d 的文件:
./val/val_0157.jpg
./val/val_0651.jpg

SHA-256哈希值为 40253bf934d8f53c00ec0cab2af5ed88274a781645c2a534d6df7795b6272806 的文件:
./val/val_0314.jpg
./val/val_0002.jpg

SHA-256哈希值为 45599ddd16eb3607a1f2413690b92d0875b89df1b4d20739bd03d5725e8de8f2 的文件:
./val/val_0697.jpg
./val/val_0698.jpg

SHA-256哈希值为 550585508f14f6d22849e67a33c541f9eab034657b3aba8614edc67676640d79 的文件:
./val/val_0961.jpg
./val/val_0774.jpg

SHA-256哈希值为 5cb2b280b8ebac77b153e8c7979ecb5453781bdda2e8efe4a71b6ff1f2402d37 的文件:
./val/val_0552.jpg
./val/val_0655.jpg

SHA-256哈希值为 6045833822e0d593da1b9a21f709e0700db6378dc2137f37e222f39521e86e51 的文件:
./val/val_0970.jpg
./val/val_0318.jpg

SHA-256哈希值为 6d4123d07a1520d92143e66f15a43c5797250d0737bfce7bb08fa2b6729412d9 的文件:
./val/val_0948.jpg
./val/val_0082.jpg

SHA-256哈希值为 d4e31a69a98137d93288cd0324d327bc45f2a205d68a8bd3f1b9e4ffd3b49227 的文件:
./val/val_0062.jpg
./val/val_0589.jpg

SHA-256哈希值为 f9796812f3c06602d67eb6cc73379665decb2f4512e3fc27a6db3e6ac6a5a4c6 的文件:
./val/val_0451.jpg
./val/val_0199.jpg
